meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* MdJ: If Solid is a way to signal what can be done but there is no consistent way of achieving it. ...
+* JW: The CG is here to incubate specs. It should be a goal of consensus rather than specification divergance. For issue 711, that was particularly controversial, I don't think we will hit that many issues with this degree of controversy. 
+* MdJ: WAC vs. ACP, SAI, UMA.
+* JW: for those, my view that CG should be incubating both WAC and ACP. We can reach consensus and LWS WG can look at them and decide what will be put forward in LWS spec and have more contancious debate there. For SAI vs. Type indexes etc. Specs that are not in scope of LWS, we need to decide how to choose from them. This should be done in data driven maner. Seeing how the cover the functional requirements. Data interop spec should be one 
+* eP: For example with SAI it's a little bit more complex, it's not SAI vs type indexes, there is also quite a lot about authorization there, and for the LWS WG there are VCs and data grants in scope, so there is some overlap there.
+* eP: I'm happy that JW refers to requirements, like the WG, the CG needs to work with scopes and requirements. The conversations often seem to be emotional and sentimental. But we should look at which requirements each proposal satisfies.
+* JW: The data grants, I belive are mentikoned in LWS charter. In ESS consent flow VCs are used as part of access requrest mechanism. That has been put into ACP via `acp:vc`. I would have a conversation with Aaron as well as Wouter who is working on UMA flow for CSS. You could see if you can come to rought consensus. Otherwise this can split into different specs.
+* eP: ...
+* JW: I put some links in the chat about other implementations of Inrupt specific flow
+  * https://amc.inrupt.com/login?returnTo=%2F
+  * https://github.com/openwallet-foundation-labs/solid-data-wallet
+* JW: For 711 specifically it would be good to merge it.
+* MdJ: It seems that it cooled down a bit, we can move forward.
+* MdJ: I think we had some conversation of what it means to call our repos specifications. While we are incubating we should be aware of the rest of the Solid project.
+* MdJ: In the future when we make a spec change, we can add a note worning that WG can change it down the road.
+* JZ: So far there has been a very good relation between Practicioners and CG. When we have outreach, we should either do it with Practicioners or joint meeting. For example catalogue, one part would be community focued and another part like shapes fits more CG. We should work on that jointly instead of discussing outreach issues just here in the CG. For example having an Awesome list would be more of Practicioners topic. We will do our part in keeping CG involved on how we are progressing. I would appreciate if CG keeps Practiciones involved when it comes to outreach and community involvement.
```

Comment:
```suggestion
* JZ: So far there has been a very good relation between Practitioners and CG. When we have outreach, we should either do it with Practitioners or joint meeting. For example, catalogue, one part would be community focused, and another part, like shapes, fits more CG. We should work on that jointly instead of discussing outreach issues just here in the CG. For example, having an Awesome list would be more of a Practitioners topic. We will do our part in keeping CG involved on how we are progressing. I would appreciate if CG keeps Practitioners involved when it comes to outreach and community involvement.
```